### PR TITLE
ENH produce more than one resampled noise image

### LINKED ISSUE #189: make extra coadded noise images for the sims

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds the feature to produce more than one noise image for each slice.

Closes #189

### CODE REVIEW

File: pizza_cutter/des_pizza_cutter/_pizza_cutter.py [hunk 1]
```diff
@@ -122,6 +124,9 @@ def make_des_pizza_slices(
         the single epoch images. See the documentaion of
         `pizza_cutter.des_pizza_cutter._coadd_slices._build_slice_inputs`
         for details on the required entries.
+    n_extra_noise_images : int, optional
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_pizza_cutter.py:127]:
```suggestion
    n_extra_noise_images : int
```

File: pizza_cutter/des_pizza_cutter/_pizza_cutter.py [hunk 2]
```diff
@@ -358,8 +367,15 @@ def _process_results(
...
-            fits=fits, data=results["noise"],
+            fits=fits, data=results["noises"][0],
             ext=NOISE_CUTOUT_EXTNAME, start_row=start_row)
+        if len(results["noises"]) > 1:
+            for i in range(len(results["noises"])-1):
+                _write_single_image(
+                    fits=fits, data=results["noises"][i+1],
+                    ext="noise%d_cutouts" % (i+1), start_row=start_row,
+                    ext_info=NOISE_CUTOUT_EXTNAME,
+                )
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_pizza_cutter.py:378]:
```suggestion
            for i in range(1, len(results["noises"])):
                _write_single_image(
                    fits=fits, data=results["noises"][i],
                    ext="noise%d_cutouts" % i, start_row=start_row,
                    ext_info=NOISE_CUTOUT_EXTNAME,
                )
```

**Comment by beckermr** [comment]
OK @esheldon This PR is ready for review. My plan is to have DESDM make these extra images. We may not used them in the end, but having them around might be useful.

**Review by esheldon — APPROVED** [review]
looks ok to me

**Pull request merged**